packages/bits/src/lib/table/table-cell/table-cell.directive.ts

```diff
@@ -81,6 +83,16 @@ export class TableCellDirective extends CdkCell implements OnInit, OnDestroy, On
                     this.rightEdgeActive = !this.rightEdgeActive;
                 });
         }
+
+        this.tableStateHandlerService.columnWidthSubject.subscribe(() => {
+            const columnWidth = this.tableStateHandlerService.getColumnWidth(this.columnDef.name);
+
+            if (columnWidth > 45) {
+                if (this.tableStateHandlerService.getColumnWidthFixed(this.columnDef.name)) {
+                    elem.style.width = elem.style.maxWidth = elem.style.minWidth = columnWidth + "px";
```

Comment:
I think it would be good to put these assignments on separate lines. One assignment per line is less error prone as the person reading the code might miss an assignment in the middle of a line.